CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Changed
 
+- We replaced "Close" by "Close library" and placed it after "Save all" in the File tab. [#10043] (https://github.com/JabRef/jabref/pull/10043)
```

Comment:
```suggestion
- We replaced "Close" by "Close library" and placed it after "Save all" in the File menu. [#10043] (https://github.com/JabRef/jabref/pull/10043)
```

Reply:
and no space after the `[]`and the round `()`ones